src/bacnet/basic/object/netport.c

```diff
@@ -129,9 +129,7 @@ static const int Network_Port_Properties_Required[] = {
     PROP_OBJECT_TYPE,       PROP_STATUS_FLAGS,
     PROP_RELIABILITY,       PROP_OUT_OF_SERVICE,
     PROP_NETWORK_TYPE,      PROP_PROTOCOL_LEVEL,
-    PROP_NETWORK_NUMBER,    PROP_NETWORK_NUMBER_QUALITY,
```

Comment:
Should we not  keep the net no. , net no. quality and apdu_len as optional prop's ?